Cartfile.resolved

```diff
@@ -0,0 +1,2 @@
+binary "https://downloads.localytics.com/SDKs/iOS/Localytics.json" "5.2.0"
```

Comment:
Typically we avoid checking in generated artifacts, although this file in particular I haven't added to .gitignore everywhere... perhaps we could delete this and also update the `.gitignore` file? (You can also add `*.framework.zip` and delete the two `.DS_Store` files)